src/main/java/reposense/authorship/AuthorshipReporter.java

```diff
@@ -4,7 +4,13 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-import reposense.dataobject.RepoConfiguration;
+import reposense.authorship.aggregator.FileResultAggregator;
+import reposense.authorship.analyzer.FileInfoAnalyzer;
+import reposense.authorship.extractor.FileInfoExtractor;
```

Comment:
The 3 main classes is fine to leave it in the main package itself, since they have the highest hierarchical order.

Reply:
Done